ocs_ci/ocs/cluster.py

```diff
@@ -903,6 +904,84 @@ def get_osd_utilization():
     return osd_filled
 
 
+def get_ceph_df_detail():
+    """
+    Get ceph osd df detail
+    Returns:
+         dict returned from cmd
+         
+    """
+    ceph_cmd="ceph df detail"
+    ct_pod = pod.get_ceph_tools_pod()
+    output = ct_pod.exec_ceph_cmd(ceph_cmd=ceph_cmd)
+    return output
+
+
+def validate_replica_data(pool_name, replica):
+    """
+    Check if data is replica 2 or 3
+    Args:
+        replica (int): size of the replica(2,3)
+        pool_name: name of the pool to check replica
+
+    Returns:
+        True if replicated data size is meet rep config and False if dont
```

Comment:
bool: True if the replicated data size is meet rep config and False otherwise

Reply:
Done